source/getting-started.txt

```diff
@@ -0,0 +1,135 @@
+.. _app-services-get-started:
+
+================
+Getting Started
+================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Tutorials </tutorials>
+   Template Apps </template-apps>
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _app-services-get-started-overview:
+
+The resources on this page are designed to help you explore Atlas App 
+Services. To explore a working codebase that showcases App Services' 
+features, check out a Template App. For a more guided experience in
+learning how to develop with App Services and Realm, start with a 
+Tutorial.
+
+.. _app-services-get-started-template-apps:
+
+Template Apps
+-------------
+
+Bootstrap App Services with a :doc:`Template App </template-apps>`. Template apps bring together
+many of the building blocks available in Atlas App Services and start you 
+off with a prebuilt app that you can customize. The following template
+apps are available:
+
+- A mobile todo app written with Realm SDKs that syncs to an App Services App
+- A hosted todo web app using the GraphQL API or Realm Web SDK
+- An Event-driven :doc:`Database Trigger </triggers/database-triggers>` 
+  template that updates a view in a separate collection.
+
+Template apps are available for the following languages:
+
+- JavaScript (React Native)
+- Kotlin (Android)
+- Swift (iOS + SwiftUI)
+- ...and more!
+
+Checkout the template apps or visit the Tutorials section for a walkthrough
+of a template app.
+
+.. button:: Check out the Template Apps
+   :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_realm_template_apps
+
+.. _app-services-get-started-tutorials:
+
+Tutorials
+---------
+
+Unlike Template Apps, which provide a pre-built app to explore by 
+yourself, Tutorials provide a detailed step-by-step guide to developing 
+apps that use App Services features.
+
+.. card-group::
+   :style: extra-compact
+   :columns: 2
+
+   .. card::
+      :headline:  iOS with SwiftUI Template App Tutorial
+      :url: https://www.mongodb.com/docs/realm/tutorial/swiftui/ 
+      :icon: /images/icons/swift_sdk.svg
+      :icon-alt: Swift SDK
+      
+      Learn how to develop an iOS app that seamlessly syncs data using the
+      Realm Swift SDK and Device Sync. 
+
+   .. card::
+      :headline:  Android with Kotlin Template App Tutorial
+      :url: https://www.mongodb.com/docs/realm/tutorial/kotlin/ 
+      :icon: /images/icons/kotlin_sdk.svg
+      :icon-alt: Kotlin SDK
+      
+      Learn how to develop a Android app that seamlessly syncs data using 
+      the Realm Kotlin SDK and Device Sync. 
```

Comment:
```suggestion
      Learn how to develop a Android app that syncs data using 
      the Realm Kotlin SDK and Device Sync. 
```

nit: i'd remove word seamlessly here, and in the following device sync tutorial descriptions. doesn't really add anything